src/core/expression/qgsexpressionfunction.cpp

```diff
@@ -7728,6 +7728,12 @@ typedef bool ( QgsGeometry::*RelationFunction )( const QgsGeometry &geometry ) c
 static QVariant executeGeomOverlay( const QVariantList &values, const QgsExpressionContext *context, QgsExpression *parent, const RelationFunction &relationFunction, bool invert = false, double bboxGrow = 0, bool isNearestFunc = false, bool isIntersectsFunc = false )
 {
 
+  if ( ! context )
+  {
+    parent->setEvalErrorString( QObject::tr( "This function was called without an expression context." ) );
```

Comment:
We probably don't need to translate this -- this situation should never arise in a user-visible context